Makefile

```diff
@@ -42,12 +43,17 @@ rapidwright:
 	cd third_party/RapidWright && make
 
 install_fasm2bels:
-	git submodule init
-	git submodule update
+	git submodule init third_party/fasm2bels
+	git submodule update third_party/fasm2bels
```

Comment:
I know this isn't your code, but since we have multiple submodules we're initializing, we should probably add another target, say `submodules`, that runs `git submodule init` and `git submodule update` for everything.  Also, I'm not as familiar with pip install as @reillymck, but do we need to both `pip install` wafove and run `git pull && make`?  At first glance it seems that one or the other would be sufficient.